api/models/helpers.go

```diff
@@ -113,7 +110,11 @@ var idAlphabet = []rune("ABCDEFGHIJKLMNOPQRSTUVWXYZ")
 func generateId(prefix string, size int) string {
 	b := make([]rune, size)
 	for i := range b {
-		b[i] = idAlphabet[rand.Intn(len(idAlphabet))]
+		idx, err := rand.Int(rand.Reader, big.NewInt(int64(len(idAlphabet))))
+		if err != nil {
+			panic(err)
```

Comment:
I don't love this, but in practice I think it's vanishingly rare; would you prefer I added the possibility of the error return to the signature and altered the callers to propogate?

Could also introduce a new generateSecurePassword function, but I think that's more trouble than it's worth given that there's no harm in using a CSPRNG everywhere.